lib/wallet/wallet.js

```diff
@@ -1629,12 +1629,14 @@ class Wallet extends EventEmitter {
    * @param {String} name
    * @param {Number|String} acct
    * @param {MTX?} mtx
+   * @param {Address?} addr
    * @returns {Promise<MTX>}
    */
 
-  async makeOpen(name, acct, mtx) {
+  async makeOpen(name, acct, mtx, addr) {
```

Comment:
If `addr` is provided then `acct` has no use.

Reply:
Hm true. But `acct` is mandatory and also matches other `make*` methods signatures. Not sure what can be done